src/styles/blocks/_current-events.scss

```diff
@@ -0,0 +1,141 @@
+.current-events {
+
+  &__card {
+    &:not(:last-child) {
+      margin-bottom: 60px;
+
+      @include onTablet {
+        margin-bottom: 64px;
+      }
+
+      @include onDesktop {
+        margin-bottom: 104px;
+      }
+    }
+
+    @include onDesktop {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+    }
+  }
+
+  &__photo img {
+    object-fit: cover;
+    object-position: center;
+    width: 280px;
+    height: 210px;
+    margin-bottom: 20px;
+    transition: all 0.3s ease;

```

Comment:
don't use it like that, because it's the same as transition: all 0.3s
It's too hard for the browser
list all property
<img width="1085" alt="image" src="https://user-images.githubusercontent.com/91411692/223972635-55991886-1eab-4cc7-a18a-84b887f9f9d5.png">